x-pack/plugins/infra/server/lib/domains/log_entries_domain/builtin_rules/filebeat_apache2.test.ts

```diff
@@ -12,12 +12,12 @@ describe('Filebeat Rules', () => {
   test('Apache2 Access', () => {
     const event = {
       'apache2.access': true,
-      'apache2.access.remote_ip': '192.168.1.42',
-      'apache2.access.user_name': 'admin',
-      'apache2.access.method': 'GET',
-      'apache2.access.url': '/faqs',
-      'apache2.access.http_version': '1.1',
-      'apache2.access.response_code': '200',
+      'source.ip': '192.168.1.42',
+      'user.name': 'admin',
+      'http.request.method': 'GET',
+      'url.original': '/faqs',
+      'http.version': '1.1',
+      'http.response.status_code': '200',
       'apache2.access.body_sent.bytes': 1024,
```

Comment:
This field changed to `http.request.body.bytes`

Reply:
I fixed those fields in a PR to this branch.